### ISSUE TITLE
Sort Transformer - description/UI 

### ISSUE THREAD

**Issue by schanzer** [P-bibs/codap-transformers#224]
- [ ] What’s the difference between a “key expression” and a “formula expression”? If there is one, can it be explained here?
- [ ] What’s needed is not really an expression at all - it’s an attribute. 
- [ ] Also, the Domain of Sort ought to be “Table" -- but "row" is provided.

**Comment by thomascastleman** [comment]
From conversation with @schanzer:

We want a transformer that sorts by attribute (matches the `order-by` used in Pyret better). Should we add an `Order By` or `Sort By` transformer that sorts by a particular attribute? (and keep the original expression-based sort as well)

@shriram thoughts?

**Comment by shriram** [comment]
Does it make sense to have two, or fold the two into one? I worry about how people will make sense of two transformers that have similar names. "Order" and "Sort" don't immediately tell me "this is by attribute, that is by expression"…

**Comment by thomascastleman** [comment]
I agree that "Order" and "Sort" probably aren't the names we should use to differentiate those two.

I think the idea is to provide a simpler transformer (by attribute) that matches Pyret and is less easy to mess up, while also providing a more complex version (by expression) for when they're ready for that. That said, we could try making this selectable through a dropdown which would then reveal/hide parts of the UI. The default setting could be "by attribute" to make the common case simple.

**Comment by schanzer** [comment]
@thomascastleman ooh, I like that idea very much. Go with it.

**Comment by shriram** [comment]
Yep, that's what I think I would prefer, too!

**Comment by retabak** [comment]
Finally reading through these issues. I like this improvement a lot!
